modules/plugin/geopkg/src/main/java/org/geotools/geopkg/GeoPackage.java

```diff
@@ -345,6 +345,16 @@ static void createFunctions(Connection cx) throws SQLException {
             cx = ((DelegatingConnection) cx).getDelegate();
         }
 
+        try (var stmt = cx.createStatement()) {
+            try (var rs =
+                    stmt.executeQuery(
+                            "select exists(select 1 from pragma_function_list where name='ST_MinX')")) {
+                if (rs.getInt(1) == 1) {
+                    return; // already exist
+                }
+            }
+        }
+
```

Comment:
This seems to be unrelated to the DATETIME management, but came from an attempt at upgrading to newer versions of SQLite, that don't accept a second registration of the same function, right? Nevertheless, does not hurt.